ckanext/ontario_theme/templates/internal/organization/bulk_process.html

```diff
@@ -0,0 +1,23 @@
+{% ckan_extends %}
+
+{% block search_form %}
+  {% if request['params'] and 'sort' in request['params'] %}
+    {% set sort_by_selected = request.params['sort'] %}
+  {% else %}
+    {% set sort_by_selected = "titles asc" %}
+  {% endif %}
+  {% set sorting = [
+      (_('Name Ascending'), 'titles asc'),
+      (_('Name Descending'), 'titles desc'),
+      (_('Last Modified'), 'data_modified desc') ] %}
+  {% set extras = {'id': c.page.items[0]['organization']['id']} %}
```

Comment:
```suggestion
  {% set extras = {'id': group_dict.id} %}
```
Core ckan has this `group_dict.id` for the organization id handy that you can use.